src/pages/data-reporting/reports-api/v3.4/index.mdx

```diff
@@ -414,6 +430,62 @@ https://api.livechatinc.com/v3.4/reports/chats/ranking \
 </Code>
 </Section>
 
+## Agents
+
+<Section>
+<Text>
+
+### Availability
+
+Shows for how long an agent, group, or the whole account was available for chatting during a specified period of time.
+
+#### Specifics
+
+|                     |                                                                    |
+| ------------------- | -------------------------------------------------------------------|
+| **Method URL**      | `https://api.livechatinc.com/v3.4/reports/agents/availability`     |
+| **HTTP method**     | POST, GET                                                          |
+| **Required scopes** | -                                                                  |
```

Comment:
it requires some scope, isn't it?

Reply:
yes, I don't know how I have copied this without it...

Reply:
added `reports_read` scope